src/App/Fossa/Container.hs

```diff
@@ -53,44 +44,39 @@ dispatch ::
   ContainerScanConfig ->
   m ()
 dispatch = \case
-  AnalyzeCfg cfg ->
-    if usesExperimentalScanner cfg
-      then AnalyzeNative.analyzeExperimental cfg
-      else Analyze.analyze cfg
+  AnalyzeCfg cfg -> do
+    if (usesExperimentalScanner cfg)
+      then
+        logWarn $
+          vsep
+            [ "DEPRECATION NOTICE"
+            , ""
+            , "Experimental scanner has been PROMOTED as default scanner. By default, FOSSA CLI uses"
+            , "this native container scanner (previously known as experimental scanner)."
+            , ""
+            , "In future, using --experimental-scanner may yield fatal error. Please stop"
+            , "using --experimental-scanner flag for container scanning."
```

Comment:
[nit]: Use authoritative tone instead of suggestion:
```suggestion
            , "The --experimental-scanner flag is now deprecated, and has no effect." 
            , "In the future, using this flag will cause a fatal error."
            , "To avoid these errors, remove the flag from your fossa commands."
```